va/va.h

```diff
@@ -1172,6 +1211,36 @@ typedef union _VAConfigAttribValContextPriority{
 #define VA_PROCESSING_RATE_ENCODE                     0x00000001
 /** \brief Driver supports decode processing rate report  */
 #define VA_PROCESSING_RATE_DECODE                     0x00000002
+
+/** \brief Attribute value for VAConfigAttribDisplayLinkHdcpWired and VAConfigAttribDisplayLinkHdcpWireless*/
+typedef union _VAConfigAttribValProtectedDisplayLink {
+    struct {
+        /** \brief session types - None = 0, Type0 = 1, Type1 = 2*/
+        uint32_t session_type                   : 3;
+        /** \brief HDCP protocol version - 1.4 = 0, 2.2 = 1, 2.3 = 2*/
+        uint32_t protocol_version               : 5;
+        /** \brief connector id, refer uapi drm_mode.h for connector id definition */
+        uint32_t connector_id                   : 5;
+        /** \brief reserved bits for future, must be zero */
+        uint32_t reserved                       : 19;
+    } bits;
+    uint32_t value;
+} VAConfigAttribValProtectedDisplayLink;
+
+/** \brief Session type for VAConfigAttribValProtectedDisplayLink */
```

Comment:
could these definition could be OR'd together?

Reply:
Remove this structure since it will not be used.